app/webpacker/styles/icons.scss

```diff
@@ -134,6 +134,13 @@
   height: 30px;
 }
 
+.icon-mail-black {
```

Comment:
I know this is included as an icon rather than an image but does it need to be marked as decorative @ethax-ross ?